DOCSP-39726: Usage Example that describe query (#114)
mongodb/docs-kotlin: source/usage-examples/updateMany.txt

Example
-------

In this example, we use a ``Filter`` builder to filter our query for
movies in the genre "Frequently Discussed". 

> We then update documents that match our query in the ``movies`` collection of the

Reviewer: ```suggestion
Next, update documents that match our query in the ``movies`` collection of the
```
